internal/identity/providers.go

```diff
@@ -107,6 +110,15 @@ func (p *Provider) GetSignInURL(state string) string {
 	return p.oauth.AuthCodeURL(state, oauth2.AccessTypeOffline)
 }
 
+// check if provider has info endpoint, try to hit that and gather more info
+// especially useful if initial request did not an contain email, or subject
+// https://openid.net/specs/openid-connect-core-1_0.html#UserInfo
+var claims struct {
```

Comment:
Can we avoid using global state to store the returned claims? Though we _should_ only be setting this once on initialization, I want to avoid any hidden state issues.

Reply:
Yeah, I also had this in mind but I wanted to know what you think about the current implementation.

I'll work on it, thanks.